app/controllers/organizations/signal_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Organizations
+  class SignalController < ApplicationController
+    def captcha_form; end
+
+    def register
+      response = SignalAdapter::RegisterPhoneNumberService.new(organization_id: @organization.id, register_data: register_data).call
+      case response
```

Comment:
```suggestion
      case SignalAdapter::RegisterPhoneNumberService.call(organization_id: @organization.id, register_data: register_data)
```

Reply:
This doesn't account for error handling where we want to pass the response to the error handler.